src/mcpbr/benchmarks/bigbench_hard.py

```
@@ -0,0 +1,230 @@
+
+        all_tasks: list[dict[str, Any]] = []
+        for subtask in subtasks:
+            try:
+                ds = load_dataset(self.dataset, subtask, split="test")
+                for idx, item in enumerate(ds):
+                    task = dict(item)
+                    task["subtask"] = subtask
+                    task["instance_id"] = f"bbh_{subtask}_{idx}"
+                    all_tasks.append(task)
+            except Exception:
+                continue
```

Ruff (0.14.14)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)